common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -74,7 +74,9 @@ public enum Operation {
         BROKER_READ_RESTRICTIONS_MANAGER(API.READ_RESTRICTIONS_MANAGER, null),
         MSAL_GET_PREFERRED_AUTH_METHOD(API.GET_PREFERRED_AUTH_METHOD, null),
         BROKER_INDIVIDUAL_LOGS_UPLOAD(API.BROKER_INDIVIDUAL_LOGS_UPLOAD, null),
-        BROKER_API_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS(API.BROKER_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS, null);
+        BROKER_API_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS(API.BROKER_RESTORE_MSA_ACCOUNTS_WITH_TRANSFER_TOKENS, null),
+        BROKER_WEBAPPS_API_GET_SUPPORTED_WEB_APPS_CONTRACTS(API.WEBAPPS_GET_SUPPORTED_WEB_APPS_CONTRACTS, null),
+        BROKER_WEBAPPS_API_EXECUTE_WEB_APPS_REQUEST(API.WEBAPPS_EXECUTE_WEB_APPS_REQUEST, null);
```

Comment:
Should this rather be named
MSAL_WEBAPPS_API_GET_SUPPORTED_WEB_APPS_CONTRACTS
MSAL_WEBAPPS_API_EXECUTE_WEB_APPS_REQUEST

or just
WEBAPPS_API_GET_SUPPORTED_WEB_APPS_CONTRACTS,
WEBAPPS_API_EXECUTE_WEB_APPS_REQUEST

given these are meant to serve client app (MSAL side) requests.

Reply:
Using the prefix. BROKER_WEBAPPS_API_